test_all.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+./gradlew test "$@"
```

Comment:
A bit odd that the arguments are passed particularly to this one command. I would just hardcode `--no-daemon` here.

Reply:
We don't want `--no-daemon` when running locally. This script is just a simple wrapper, I don't think such argument passing is a big issue